src/main.js

```diff
@@ -1,12 +1,14 @@
-const { app, nativeImage, BrowserWindow } = require('electron')
+const { app, nativeImage, BrowserWindow, Tray, Menu } = require('electron')
 const BadgeGenerator = require('./badge_generator')
 const REFRESH_RATE = 5000; // 5 seconds
+const iconPNG = "1024px-Google_Voice_icon_(2020).png";
 const icon = nativeImage.createFromPath(
-    app.getAppPath() + "/public/1024px-Google_Voice_icon_(2020).png"
```

Comment:
Need to create a new 16x16 image and use that one for the tray, but keep the big one for the dock / task bar